examples/plot_connectivity.py

```diff
@@ -52,9 +51,14 @@
 ###############################################################################
 # We can modify the connectivity list to test the effect of different
 # connectivity patterns. For example, we can remove all layer 2 inhibitory
-# connections.
+# connections. In the default network, the src_gids of each connection are
+# all of the same type so we'll just check the first element. Connections are
+# stored under ``conn['gid_pairs']`` as a list of lists of the form
+# ``[[src_gid, [target_gid1, target_gid1]], ...]``. For each inner list,
+# the first element corresponds to the src_gid, and the second corresponds to
+# all of its target cells.
```

Comment:
I'm almost tempted to suggest removing this part of the example. Would you be okay with that? 

We need to think a little more how to modify existing connectivity. I'm not sure we want to give a complicated example to users with double indexing even if there is a nice explanation! We can revisit this in a later PR. The API is nicely developing though!

Reply:
Agreed the modifications still require "hard coding". Perhaps a better interface will involve a function that also accepts sources and targets just like the current API, and clears all connections between those gids.

Reply:
or maybe a parameter `reset_old_values` or `overwrite` ...

but let's go one PR at a time. I think this PR is shaping up really well and API is improving. If you can add the appropriate tests, make things a little high-level I requested and bring things to MRG, we can continue iterating in another PR!